### PR TITLE
Improve the Helm charts docs

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

File: helm-chart/kubray-operator/README.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 Please use command below:
 ```console
-$ helm install  kubray-operator . --values values.yaml --namespace default --create-namespace
+$ helm install kubray-operator . --values values.yaml --namespace kubray-operator --create-namespace
```
Comment (Jeffwan) [inline helm-chart/kubray-operator/README.md:21]: typo.  `kubray` -> `kuberay`. 

In other examples, we proposed to use namespace `ray-system`.   Besides the operator, we may have other components like kuberay apiserver or node detector later. I think it's better to use a different name than `kuberay-operator`.  what do you think?

Comment (haoxins) [reply helm-chart/kubray-operator/README.md]: > typo. `kubray` -> `kuberay`.

Fixed.

> But why the dir is named `kubray-operator`?

---

> In other examples, we proposed to use namespace ray-system. Besides the operator, we may have other components like kuberay apiserver or node detector later. I think it's better to use a different name than kuberay-operator. what do you think?

I used `ray-system` in the other (ray cluster) examples.

---

BTW, it's better to remove `kube` or `kub` prefix IMO.

Prefixed with `kube` is sometimes the K8s system's privileges.

**Comment by Jeffwan** [comment]
The change looks good to me

**Review by Jeffwan — COMMENTED** [review]
The change looks good to me

**Pull request merged**